sass/components/html/page-header-complete.html

```diff
@@ -0,0 +1,54 @@
+<div class="page-header">
+  <nav class="page-navigation" role="navigation">
+    <div class="container-fluid">
+      <nav class="breadcrumb breadcrumb-processed breadcrumb--collapsible" role="navigation" aria-label="breadcrumbs">
+        <ol class="breadcrumb__segments-wrapper">
+          <li class="breadcrumb__segment breadcrumb__segment--first">
+            <a href="#" class="breadcrumb__link is-internal" rel="home">
+              <span class="breadcrumb__text">European Commission</span>
+            </a>
+          </li>
+          <li class="breadcrumb__segment breadcrumb__segment--last">
+            <a href="#" class="breadcrumb__link">
+              <span class="breadcrumb__text">Digital single market</span>
+            </a>
+          </li>
+          <li class="breadcrumb__segment breadcrumb__segment--last">
+            <a href="#" class="breadcrumb__link">
+              <span class="breadcrumb__text">Strategy</span>
+            </a>
+          </li>
+           <li class="breadcrumb__segment breadcrumb__segment--last">
+            <a href="#" class="breadcrumb__link">
+              <span class="breadcrumb__text">Newsroom</span>
+            </a>
+          </li>
+        </ol>
+      </nav>
+    </div>
+  </nav>
+  <div class="container-fluid page-header__site-identification">
+    <h3>Digital single market</h3>
+  </div>
+  <div class="container-fluid page-header__hero-title">
+    <div class="row padding-reset">
```

Comment:
this is not anything i've been changing so, no, i would leave it as such. You can create a ticket for this if you think it has to be removed